package.json

```diff
@@ -18,8 +18,8 @@
     "Opera 12.1"
   ],
   "engines": {
-    "node": ">= 8.9.1",
-    "npm": ">= 5.5.1"
+    "node": ">= 10.4.0",
```

Comment:
well now I guess we should do `10.4.1` after the security update?

Reply:
Yup, I will. The >= will accept that as well, so we probably only need to update engines again after the next Major version bump.

Reply:
I'll keep it here since https://github.com/Automattic/wp-calypso/pull/25469 hasn't landed yet. but I don't think this will be an issue.